packages/tools/fetch-tool/package.json

```diff
@@ -64,6 +63,7 @@
 		"typescript": "~5.4.5"
 	},
 	"typeValidation": {
+		"disabled": true,
```

Comment:
What is this about? Can we call this out in the PR description?

Reply:
I see -previous dep is removed above. Please separate or call out.

Reply:
Removed.